tools/ami-cleanup/workflows/cd.yaml

```diff
@@ -0,0 +1 @@
+../../../.github/workflows/ami-cleanup-cd.yaml
```

Comment:
Should these symlinks go the other way?  The originals in `tools/` and the symlink in `.github/`?

https://github.com/gravitational/shared-workflows/blob/d8562e07c65a15c5da4472f6d1601f5ea6e9bedd/rfd/0001-tooling-requirements.md#L61-L62

Reply:
Unfortunately no. I just had a discussion with Github support about this (after the workflows initially failed to run) and GHA doesn't support symlinks in the workflows directory. I got a very noncommittal "we'll put it in a file somewhere and think about it" type response when I asked about adding this support.

I realize that this is in contradiction to the RFD, but unfortunately it has to be this way if we want any sort of direct workflow/project linkage.

Reply:
Thats fine.  Maybe we should update the RFD then?

Reply:
Yea I was thinking about that. I don't think we often update RFDs but I think it is reasonable to do so in this case.